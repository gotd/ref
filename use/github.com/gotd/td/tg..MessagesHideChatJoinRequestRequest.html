<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/gotd/td/tg.MessagesHideChatJoinRequestRequest</title>
<link href="../../../../css/light-v0.5.6.css" rel="stylesheet">
<script src="../../../../jvs/golds-v0.5.6.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;">type <b><a href="../../../../pkg/github.com/gotd/td/tg.html">github.com/gotd/td/tg</a>.<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-41">MessagesHideChatJoinRequestRequest</a></b></span>

<span class="title">25 uses</span>

	github.com/gotd/td/tg (current package)
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-41">tl_messages_hide_chat_join_request_gen.go#L41</a>: type <b>MessagesHideChatJoinRequestRequest</b> struct {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-66" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-66">#L66</a>: 	_ bin.Encoder     = &amp;<b>MessagesHideChatJoinRequestRequest</b>{}
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-67" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-67">#L67</a>: 	_ bin.Decoder     = &amp;<b>MessagesHideChatJoinRequestRequest</b>{}
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-68" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-68">#L68</a>: 	_ bin.BareEncoder = &amp;<b>MessagesHideChatJoinRequestRequest</b>{}
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-69" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-69">#L69</a>: 	_ bin.BareDecoder = &amp;<b>MessagesHideChatJoinRequestRequest</b>{}
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-72" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-72">#L72</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) Zero() bool {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-93" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-93">#L93</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) String() string {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-97" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-97">#L97</a>: 	type Alias <b>MessagesHideChatJoinRequestRequest</b>
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-102" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-102">#L102</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) FillFrom(from interface {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-115" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-115">#L115</a>: func (*<b>MessagesHideChatJoinRequestRequest</b>) TypeID() uint32 {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-120" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-120">#L120</a>: func (*<b>MessagesHideChatJoinRequestRequest</b>) TypeName() string {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-125" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-125">#L125</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) TypeInfo() tdp.Type {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-153" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-153">#L153</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) SetFlags() {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-160" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-160">#L160</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) Encode(b *bin.Buffer) error {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-169" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-169">#L169</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) EncodeBare(b *bin.Buffer) error {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-193" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-193">#L193</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) Decode(b *bin.Buffer) error {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-204" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-204">#L204</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) DecodeBare(b *bin.Buffer) error {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-232" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-232">#L232</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) SetApproved(value bool) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-243" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-243">#L243</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) GetApproved() (value bool) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-251" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-251">#L251</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) GetPeer() (value InputPeerClass) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-259" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-259">#L259</a>: func (h *<b>MessagesHideChatJoinRequestRequest</b>) GetUserID() (value InputUserClass) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-279" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-279">#L279</a>: func (c *Client) MessagesHideChatJoinRequest(ctx context.Context, request *<b>MessagesHideChatJoinRequestRequest</b>) (UpdatesClass, error) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_registry_gen.go.html#line-4520">tl_registry_gen.go#L4520</a>: 		MessagesHideChatJoinRequestRequestTypeID:                                func() bin.Object { return &amp;<b>MessagesHideChatJoinRequestRequest</b>{} },
		<a href="../../../../src/github.com/gotd/td/tg/tl_server_gen.go.html#line-5049">tl_server_gen.go#L5049</a>: func (s *ServerDispatcher) OnMessagesHideChatJoinRequest(f func(ctx context.Context, request *<b>MessagesHideChatJoinRequestRequest</b>) (UpdatesClass, error)) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_server_gen.go.html#line-5051" class="path-duplicate">tl_server_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_server_gen.go.html#line-5051">#L5051</a>: 		var request <b>MessagesHideChatJoinRequestRequest</b>
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/apps-and-libs/golds.html"><b>Golds</b></a> <i>v0.5.6</i>. (GOOS=linux GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>