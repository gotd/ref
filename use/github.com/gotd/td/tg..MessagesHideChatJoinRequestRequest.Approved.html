<!DOCTYPE html>
<html>
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1">
<title>References: github.com/gotd/td/tg.MessagesHideChatJoinRequestRequest.Approved</title>
<link href="../../../../css/auto-v0.6.7.css" rel="stylesheet">
<script src="../../../../jvs/golds-v0.6.7.js"></script>
<body onload="onPageLoad()"><div>

<pre><code><span style="font-size:x-large;"><b><a href="../../../../pkg/github.com/gotd/td/tg.html">github.com/gotd/td/tg</a>.<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-41">MessagesHideChatJoinRequestRequest</a>.<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-51">Approved</a></b></span><span style="font-size: large;"><i> (field)</i></span>

<span class="title">7 uses</span>

	github.com/gotd/td/tg (current package)
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-51">tl_messages_hide_chat_join_request_gen.go#L51</a>: 	<b>Approved</b> bool
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-79" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-79">#L79</a>: 	if !(h.<b>Approved</b> == false) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-107" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-107">#L107</a>: 	h.<b>Approved</b> = from.GetApproved()
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-154" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-154">#L154</a>: 	if !(h.<b>Approved</b> == false) {
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-213" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-213">#L213</a>: 	h.<b>Approved</b> = h.Flags.Has(0)
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-235" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-235">#L235</a>: 		h.<b>Approved</b> = true
		<a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-238" class="path-duplicate">tl_messages_hide_chat_join_request_gen.go</a><a href="../../../../src/github.com/gotd/td/tg/tl_messages_hide_chat_join_request_gen.go.html#line-238">#L238</a>: 		h.<b>Approved</b> = false
</code></pre><pre id="footer">
<table><tr><td><img src="../../../../png/go101-twitter.png"></td>
<td>The pages are generated with <a href="https://go101.org/apps-and-libs/golds.html"><b>Golds</b></a> <i>v0.6.7</i>. (GOOS=linux GOARCH=amd64)
<b>Golds</b> is a <a href="https://go101.org">Go 101</a> project developed by <a href="https://tapirgames.com">Tapir Liu</a>.
PR and bug reports are welcome and can be submitted to <a href="https://github.com/go101/golds">the issue list</a>.
Please follow <a href="https://twitter.com/go100and1">@Go100and1</a> (reachable from the left QR code) to get the latest news of <b>Golds</b>.</td></tr></table></pre>